app/src/main/java/com/github/globant/githubsubscribers/subscribersdetail/view/SubscriberDetailFragment.java

```diff
@@ -69,23 +74,24 @@ private void loadArgs() {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
-        View viewFragment = inflater.inflate(R.layout.fragment_subscribers_detail, container, false);
-        presenter = new SubscriberDetailPresenterImpl(this);
-
-        profileImage = (ImageView) viewFragment.findViewById(R.id.img_avatar_subscriber_detail);
-        profileFullName = (TextView) viewFragment.findViewById(R.id.txt_full_name_subscriber_detail);
-        profileFullName.setOnClickListener(this);
-        profileUserName = (TextView) viewFragment.findViewById(R.id.txt_user_name_subscriber_detail);
-        profileCompany = (TextView) viewFragment.findViewById(R.id.txt_company_subscriber_detail);
-        profileLocation = (TextView) viewFragment.findViewById(R.id.txt_location_subscriber_detail);
-        profileFollowingCounter = (TextView) viewFragment.findViewById(R.id.txt_following_counter_subscriber_detail);
-        profileFollowersCounter = (TextView) viewFragment.findViewById(R.id.txt_followers_counter_subscriber_detail);
-        profileReposCounter = (TextView) viewFragment.findViewById(R.id.txt_repos_counter_subscriber_detail);
-
-        RecyclerView recyclerViewSubscribers = (RecyclerView) viewFragment.findViewById(R.id.list_repositories);
-        recyclerViewSubscribers.setLayoutManager(new LinearLayoutManager(getActivity()));
-        recyclerViewSubscribers.setAdapter(repositoriesAdapter);
-        repositoriesAdapter.setClickListener(this);
+        if (savedInstanceState == null) {
```

Comment:
Yes, also at that point of lifeCycle of the fragment, the view isn't created yet.